torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1644,6 +1644,21 @@ def sample_inputs_broadcast_to(op_info, device, dtype, requires_grad, **kwargs):
             make_tensor(size, device, dtype, low=None, high=None, requires_grad=requires_grad),
             args=(shape,)) for size, shape in test_cases)
 
+def sample_inputs_bitwise_and(op_info, device, dtype, requires_grad, **kwargs):
```

Comment:
Instead of a custom sample input function I think this can use

https://github.com/pytorch/pytorch/blob/10f372601dc27ccd9d1a7eaed2dbbb04fb20f962/torch/testing/_internal/common_methods_invocations.py#L1205

fyi @pmeier